src/Mustaches.test.js

```diff
@@ -161,6 +192,35 @@ describe('mustaches', () => {
     expect(placeholders).toMatchSnapshot()
   })
 
+  test('discovery with strict mode: Undefined variable', async () => {
```

Comment:
For consistency, I would name the test: "discovery with strict mode - undefined variable"